README.md

```diff
@@ -109,3 +109,13 @@ The following default Rails gems have been removed:
 
 * [coffee-rails](https://github.com/rails/coffee-rails)
 * [jbuilder](https://github.com/rails/jbuilder)
+
+## Deployment
```

Comment:
Do you instead just want to configure an `app.json` file and be done with it?

https://devcenter.heroku.com/articles/app-json-schema#buildpacks